### PR TITLE
Allow multiple fixture paths during bundle configuration.

### PR DESCRIPTION
**Pull request by wRLSS**
hautelook#404
Allow multiple `fixture_path` entries during bundle configuration.
Keeping BC-Break with previous Configuration.

### CODE REVIEW

File: src/Locator/EnvDirectoryLocator.php [hunk 1]
```diff
@@ -70,27 +70,28 @@ function (BundleInterface $bundle) use ($environment): array {
...
 
-        if (false === $path || false === file_exists($path)) {
-            return [];
-        }
+            if (false === $fullPath || false === file_exists($fullPath)) {
+                continue;
+            }
 
-        $files = SymfonyFinder::create()->files()->in($path)->depth(0)->name('/.*\.(ya?ml|php)$/i');
+            $files = SymfonyFinder::create()->files()->in($fullPath)->depth(0)->name('/.*\.(ya?ml|php)$/i');
```
Comment (theofidry) [inline src/Locator/EnvDirectoryLocator.php]: wouldn't it be better to collect the full paths first and do only one lookup with the Finder?

Comment (wRLSS) [reply src/Locator/EnvDirectoryLocator.php]: Sure, updated.

Comment (wRLSS) [reply src/Locator/EnvDirectoryLocator.php]: @theofidry

File: src/Locator/EnvDirectoryLocator.php [hunk 2]
```diff
@@ -70,25 +70,28 @@ function (BundleInterface $bundle) use ($environment): array {
...
+        $fullPaths = array_filter(array_map(function ($fixturePath) use ($environment, $path) {
+            return '' !== $environment
+                ? sprintf('%s/%s/%s', $path, $fixturePath, $environment)
+                : sprintf('%s/%s', $path, $fixturePath);
+        }, $this->fixturePaths), function ($fullPath) {
+            return $fullPath && file_exists($fullPath);
+        });
 
-        if (false === $path || false === file_exists($path)) {
+        if (empty($fullPaths)) {
```
Comment (theofidry) [inline src/Locator/EnvDirectoryLocator.php]:
```suggestion
        if ([] === $fullPaths) {
```

File: src/Locator/EnvDirectoryLocator.php [hunk 3]
```diff
@@ -70,25 +70,28 @@ function (BundleInterface $bundle) use ($environment): array {
      */
     private function doLocateFiles(string $path, string $environment): array
     {
-        $path = '' !== $environment
-            ? sprintf('%s/%s/%s', $path, $this->fixturesPath, $environment)
-            : sprintf('%s/%s', $path, $this->fixturesPath)
-        ;
-
-        $path = realpath($path);
+        $fullPaths = array_filter(array_map(function ($fixturePath) use ($environment, $path) {
```
Comment (theofidry) [inline src/Locator/EnvDirectoryLocator.php]:
```suggestion
        $fullPaths = array_filter(array_map(static function (string $fixturePath) use ($environment, $path): string {
```

**Comment by theofidry** [comment]
I can't update your branch so you'll have to do the update yourself

File: src/Locator/EnvDirectoryLocator.php (hunk 3 above)
Comment (wRLSS) [reply src/Locator/EnvDirectoryLocator.php]: UPD

File: src/Locator/EnvDirectoryLocator.php (hunk 2 above)
Comment (wRLSS) [reply src/Locator/EnvDirectoryLocator.php]: UPD

**Comment by theofidry** [comment]
Thanks!

**Pull request merged**